src/package.json

```diff
@@ -29,16 +29,15 @@
     ]
   },
   "scripts": {
-    "tsc": "tsc -skipLibCheck",
-    "build": "npm i --ignore-scripts && tsc && cd ../publish && ./pack.sh",
+    "build": "npm i && tsc",
     "tslint": "cd .. && tslint \"**/*.ts\" --config tslint.json --exclude \"**/node_modules/**\"",
-    "clean": "cd ../demo && rimraf hooks node_modules platforms && cd ../src && rimraf node_modules && npm run plugin.link",
-    "ci.tslint": "npm i --ignore-scripts && tslint '**/*.ts' --config ../tslint.json --exclude '**/node_modules/**'",
+    "clean": "cd ../demo && rimraf hooks node_modules platforms && cd ../src && rimraf node_modules && npm run build",
```

Comment:
I would expect the clean command to clean only, and not build. A rebuild command should do clean + build. Maybe we can discuss on that.